spyderlib/plugins/editor.py

```diff
@@ -944,7 +952,7 @@ def get_plugin_actions(self):
         self.recent_file_menu = QMenu(_("Open &recent"), self)
         self.recent_file_menu.aboutToShow.connect(self.update_recent_file_menu)
 

```

Comment:
Also @ccordoba12 why is the New File option in the file menu but the open last closed item in the Editor menu? I don't get the logic + I am lost in the code, have no idea how to move that menu to the Editor context